frontend/common/src/components/inputPartials/Fieldset/Fieldset.tsx

```diff
@@ -147,7 +147,7 @@ const Fieldset: React.FC<FieldsetProps> = ({
           data-h2-display="base(block)"
           data-h2-margin="base(x.125, 0, 0, 0)"
         >
-          <InputError isVisible={!!error} error={error} />
+          <InputError id={`${name}-error`} isVisible={!!error} error={error} />
```

Comment:
Hmm, good call. Should we introduce a required `id` prop here? We need some unique identifier to associate the error with the `fieldset` 🤔

Reply:
I really don't know.  :sweat_smile: I remember you used lodash uniqueid for an ID in another PR.  Maybe that could be used as a fallback?

Reply:
I just figured it out 😁 It's not great but it works 😅 

[adjust fieldset to avoid optional props](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4818/commits/a12cb1bb55f0e8f6ca2807855cfa76f9213157df)